### DISCUSSION TITLE
Is there an ETA for the custom Flight Plan Manager?

### DISCUSSION THREAD

**Discussion by ManyPandas** [flybywiresim/aircraft#4785] [General]
The title sums it up. I've always had issues with the default Flight Plan Manager built into MSFS, especially with missing waypoints on STAR procedures.

**Comment by tracernz** [comment]
From the Discord bot:
> #### When is it coming?
> Due to the community-driven nature of our projects, we do not enforce delivery timelines on our volunteer developers. We value high-quality, polished releases over frequent and quickly-released ones, which means some features might take time to land in a publicly-available build. Rest assured that our developers and contributors are working hard on making every FlyByWire project as bug-free and feature-complete as possible, but that we are not able to provide release dates or estimates for a certain feature/fix.
